lib/core.js

```diff
@@ -0,0 +1,48 @@
+
+import Mustache from "mustache"
+import fs from "fs"
+import pathModule from "path"
+
+import cleanUp from "./utils/clean_up"
+import { getFolderName, makeComponentName, createFiles, prepareFiles } from "./utils/files"
+import { addFlowOption } from "./flow"
+import { addReduxOption } from "./redux"
+
+const createComponent = (path, options) => {
+	const folderName = getFolderName(path)
+	const componentName = makeComponentName(folderName)
+
+	const componentsPath = pathModule.resolve(path, "components")
+
+	// MkdirSync recursive not working
+	fs.mkdirSync(path)
+	fs.mkdirSync(componentsPath)
+
+	let [dumbString, containerString, indexString] = prepareFiles(componentName)
+
+	let dumbOptions = {}
+	let containerOptions = {
```

Comment:
Idem, a better name for `containerOptions` could probably be `containerTags`.

Reply:
[c1ea1f2](/reacli/cli/pull/30/commits/c1ea1f257d959ef11a254cf642481f45cf4a9985)